src/modules/opengl/filter_movit_blur.cpp

```diff
@@ -28,6 +28,10 @@ using namespace movit;
 
 static int get_image( mlt_frame frame, uint8_t **image, mlt_image_format *format, int *width, int *height, int writable )
 {
+	if (!width || !height || *width < 1 || *height < 1) {
+		return 1;
```

Comment:
> So, the width and height check belongs after that call. Also, it is preferable behavior to skip the filter (with a log message) instead of return an error, because an error return will cause a white image to be created. This comment applies to all of the other filters as well.

I assumed just returning `error` as the other checks does is the correct way to do this.